src/rgw/rgw_log.cc

```diff
@@ -346,7 +346,11 @@ int rgw_log_op(RGWRados *store, RGWREST* const rest, struct req_state *s,
   else
     set_param_str(s, "REMOTE_ADDR", entry.remote_addr);
   set_param_str(s, "HTTP_USER_AGENT", entry.user_agent);
-  set_param_str(s, "HTTP_REFERRER", entry.referrer);
```

Comment:
I think this can be fixed as `HTTP_REFERER` for now and do the entire misspelling clean up as a different PR. Because after we fix the misspelling, the below logic won't be needed at all. What do you say ?